apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -235,30 +229,24 @@ export const Subscriptions: FC<SubscriptionsProps> = ({
             ),
         });
 
-    const handleCriteriaChange = ({ page }: Criteria<Subscription>) => {
-        if (page) {
-            const { index, size } = page;
-            setPageSize(size);
-            setPageIndex(index * size);
-        }
-    };
-
     return (
-        <TableWithFilter
+        <TableWithFilter<Subscription>
+            __filterQuery={filterQuery}
+            __setFilterQuery={setFilterQuery}
+            onUpdateEsQueryString={() => console.log('FILLER')}
             data={mapApiResponseToSubscriptionTableData(data)}
             tableColumns={tableColumns}
             leadingControlColumns={leadingControlColumns}
             defaultHiddenColumns={defaultHiddenColumns}
             dataSorting={dataSorting}
             pagination={pagination}
-            filterQuery={filterQuery}
-            isInvalidFilterQuery={queryContainsInvalidParts}
             isLoading={isFetching}
             localStorageKey={SUBSCRIPTIONS_TABLE_LOCAL_STORAGE_KEY}
-            onUpdateFilterQuery={setFilterQuery}
-            onCriteriaChange={handleCriteriaChange}
-            onUpdatePageSize={setPageSize}
-            onResetPageSize={() => setPageSize(defaultPageSize)}
+            onUpdatePageSize={(page) => {
```

Comment:
move this to a const function

Reply:
What do you mean with a const function?

Reply:
what I mean is to move it out of the return and into a const above it.
```
const updatePageSize => {
  if (page) {
     setPageSize(page.size);
  }
}

return (<TableWithFilter onUpdatePageSize={updatePageSize} />
```

Reply:
fixed